crates/auth/tests/tests/auth.rs

```diff
@@ -0,0 +1,57 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: Apache-2.0
+use http::{Request, StatusCode};
+use hyper::Body;
+use regex::Regex;
+use std::str;
+use tower::util::ServiceExt;
+
+use crate::routes::test_routes as app;
+use drawbridge_auth::{GITHUB, GITHUB_AUTHORIZED};
```

Comment:
Why is this not imported from `crate::`?
Also import from within the crate should come first (above the external imports)

Reply:
crate in this context doesn't point to the actual crate and only gives me /tests stuff